lib/tasks/install-blueprint.js

```diff
@@ -135,6 +140,32 @@ class InstallBlueprintTask extends Task {
     logger.info(`Loading blueprint from "${path}" ...`);
     return RSVP.resolve().then(() => Blueprint.load(path));
   }
+
+  /*
+   * NPM has 4 places it uses for .npmrc. 3 of the 4 are supported as their locations
+   * are external to the project root but if there is an .npmrc file located at the project root it will
+   * not be respected. This is because we create a tmp directory (createTempFolder) where we run
+   * the npm install. This function simply copies over the .npmrc file to the tmp directory so that it
+   * will be used during the install of the blueprint. Useful for installing private blueprints
+   * such as `ember init -b @company/company-specific-blueprint` where the module is in a different
+   * registry.
+   */
+  _copyNpmrc(tmpDir) {
```

Comment:
why is this a promise, if all the IO comands within are sync?

Reply:
Done